StringExample.java

```diff
@@ -34,8 +34,5 @@ public static void main(String[] args) {
         //String Format
         String format = String.format("The salary for abi is % 3f", 10.235000);
```

Comment:
Hi, do you mean 3 decimal places? 
If the desired output is 10.235, then the format needs to be changed to:
%.3f 
instead of % 3f